moose/build.rs

```diff
@@ -1,4 +1,6 @@
 fn main() -> Result<(), Box<dyn std::error::Error>> {
+    #[cfg(target_os = "macos")]
+    println!("cargo:rustc-link-search=native=/opt/homebrew/opt/openblas/lib");
```

Comment:
is this meant to be included?

Reply:
Yes, it adds `/opt/homebrew/opt/openblas/lib` to the linker search path (when compiling on macOS)

Reply:
neat